lms/static/sass/features/_journals.scss

```diff
@@ -129,8 +129,8 @@
       }
     }
 
-    &:before,
-    &:after {
+    &::before,
+    &::after {
```

Comment:
why we have to change these css to CSS3?

Reply:
I was getting the quality error.